extra_data/cli/make_virtual_cxi.py

```diff
@@ -27,6 +27,13 @@ def _detectors():
     return [d.__name__ for d in MPxDetectorBase.__subclasses__()]
 
 
+def cast(number:str):
```

Comment:
I liked `parse_number` better as a name for this - it's explicit about what it does.